src/main/app/forms/validation/validators/datePastConstraint.ts

```diff
@@ -0,0 +1,43 @@
+
+import {
+  registerDecorator,
+  ValidationArguments,
+  ValidationOptions,
+  ValidatorConstraint,
+  ValidatorConstraintInterface
+} from 'class-validator'
+
+import { MomentFactory } from 'common/momentFactory'
+import { LocalDate } from 'forms/models/localDate'
+
+@ValidatorConstraint()
+export class DatePastConstraint implements ValidatorConstraintInterface {
+  validate (value: any, args?: ValidationArguments) {
+    if (value === undefined) {
+      return true
```

Comment:
why not value === null || value === undefined?

if you return true in both cases.

Reply:
Yes I agree, change made